spaceship/lib/spaceship/test_flight/base.rb

```diff
@@ -0,0 +1,11 @@
+module Spaceship::TestFlight
+  class Base < Spaceship::Base
+    def self.client
+      @client ||= Client.client_with_authorization_from(Spaceship::Tunes.client)
+    end
+
+    def to_json
+      raw_data.to_json
+    end
+  end
+end
```

Comment:
Do we need to have this class now that the `TestFlight` module is inside of `Spaceship`?